.nycrc

```diff
@@ -1,10 +1,9 @@
 {
   "report-dir": "./.coverage",
   "exclude": [
-    "build/src/*{/*,/**/*}.js",
-    "build/src/*/v*/*.js",
-    "build/protos/*.js",
-    "build/test/**/*.js"
+    "src/*{/*,/**/*}.js",
```

Comment:
Eugh.  We need to configure `nyc` to do "the right thing" here for TypeScript projects.   Please file an issue in synthtool to change the template to make this work for TypeScript and for JavaScript.

Reply:
Done!